config.w32

```diff
@@ -20,9 +20,14 @@ if (PHP_MONGODB != "no") {
 
   AC_DEFINE("HAVE_MONGOC", 1);
 
-  if (!CHECK_LIB("libsasl.lib", "mongodb", PHP_MONGODB)) {
-	  WARNING("mongodb libsasl support not enabled, libs not found");
+  if (PHP_MONGODB_SASL != "no") {
+    if (CHECK_LIB("libsasl.lib", "mongodb", PHP_MONGODB)) {
+      ADD_FLAG("CFLAGS_MONGODB", " /D MONGOC_ENABLE_SASL=1");
```

Comment:
In this case, I think we still need to check for `sasl_client_done()` (as we do in `config.m4` when calling `PHP_CHECK_LIBRARY(sasl2, sasl_client_done, ...)`). I'm currently looking into whether `CHECK_FUNC_IN_HEADER()` can be used.